### PR TITLE
Primary Project Manager Table Filter Fix

### PR DESCRIPTION
**Pull request by sam-warren**
Card: https://github.com/bcgov/cas-cif/issues/1562

### CODE REVIEW

File: app/pages/cif/projects.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
         "Primary Project Managers",
         "primaryProjectManager",
         allCifUsers.edges.map((e) => e.node.fullName),
-        { allowFreeFormInput: true, sortable: false }
+        { allowFreeFormInput: false, sortable: false }
```
Comment (Sepehr-Sobhani) [inline app/pages/cif/projects.tsx:121]: We can simply remove whole `allowFreeFormInput: false` as it's false by default.

File: app/components/Table/Filters/SearchableDropdownFilter.tsx [hunk 2]
```diff
@@ -67,7 +67,6 @@ export default class SearchableDropdownFilter extends TableFilter<string> {
                   ...params.InputProps,
                   disableUnderline: true,
                   style: {
-                    fontFamily: "BC Sans",
```
Comment (Sepehr-Sobhani) [inline app/components/Table/Filters/SearchableDropdownFilter.tsx:70]: Could you kindly provide some context on why this was removed?

**Review by Sepehr-Sobhani — CHANGES_REQUESTED** [review]
Looks great 👍 just a minor change and a rebase is required!

File: app/pages/cif/projects.tsx (hunk 1 above)
Comment (sam-warren) [reply app/pages/cif/projects.tsx:121]: Good catch! Thank you

File: app/components/Table/Filters/SearchableDropdownFilter.tsx (hunk 2 above)
Comment (sam-warren) [reply app/components/Table/Filters/SearchableDropdownFilter.tsx:70]: The inclusion of this `fontFamily` property caused inconsistencies in the typeface used for the filters - part of the ticket's requirement was to have the affected filter fields' placeholder text consistent with the rest of the filter fields.

Comment (Sepehr-Sobhani) [reply app/components/Table/Filters/SearchableDropdownFilter.tsx:70]: Oh sorry, I missed that part of the ticket's context. Thank you for the clarification.

Comment (sam-warren) [reply app/components/Table/Filters/SearchableDropdownFilter.tsx:70]: All good! Thanks for your quick review :)

**Review by Sepehr-Sobhani — APPROVED** [review]

**Pull request merged**